src/lua/luafile.cc

```diff
@@ -73,6 +74,22 @@ LuaFile* subFile(LuaFile* wrapper, uint64_t start, int64_t size) {
 }
 LuaFile* uvFifo(const char* address, int port) { return new LuaFile(new PCSX::UvFifo(address, port)); }
 
+LuaServer* uvFifoListener() {return new LuaServer(new PCSX::UvFifoListener());}
```

Comment:
We also need a function to delete a LuaServer. It's not simple as it needs to schedule a stop if the server is running, and then delete in the async close callback.